develop-docs/sdk/expected-features/setup-wizards/index.mdx

```diff
@@ -53,15 +53,22 @@ For instance:
 Related to this, wizards should [check the git status](#1-check-preconditions) of the project and warn users if they're not in a clean state.
 This ensures that users can revert the changes easily if something goes wrong.
 
-### Cater to the 80%
+### Respect Users' decisions
+
+Wizards should allow users to select the high-level Sentry features (e.g. tracing, replay, profiling) they want to install.
+The SDK setup should be minimal, may be opinionated ("whatever makes sense for the platform") but 
+must respect users' decisions. 
 
-Wizards should be opinionated in which SDK options are set and cater to the 80% use case.
-The SDK setup doesn't necessarily have to be minimal.
-For example, it can include optional but recommended integrations (e.g. session replay or profiling) or options (e.g. sample rates).
+Some examples: 
+- We should not enable a feature that users previously declined (e.g. set `tracesSampleRate` if users previously declined to enable tracing)
+- We may set additional options, like the `tunnelRoute` option in Next.Js, but we should ask for consent whenever reasonable. In this exapmle, consent is important because setting this option potentially increases users' traffic and hence their infrastructure bill.
```

Comment:
```suggestion
- We may set additional options, like the `tunnelRoute` option in Next.Js, but we should ask for consent whenever reasonable. In this example, consent is important because setting this option potentially increases users' traffic and hence their infrastructure bill.
```